examples/database-seeding/README.md

```diff
@@ -0,0 +1,94 @@
+## Example Application: Database Seeding Through Parameter Configuration
+
+### Quickstart
+
+##### The 'dev' environment:
+```
+architect deploy -l arc.env.dev.yml
+```
+At http://api.localhost:3000/users, you should see an empty list of users because the AUTO_DDL parameter is set to 'migrate'.
+
+##### The 'qa' environment:
+```
+architect deploy -l arc.env.qa.yml
+```
+At http://api.localhost:3000/users, you should see a list of test users generated because the AUTO_DDL parameter is set to 'seed'.
+
+##### The 'prod' environment:
+```
+architect deploy -l arc.env.prod.yml
+```
+
+This environment is configured to connect to an existing database on `localhost:5432` (referenced by `host.docker.internal`) and has the AUTO_DDL property set to 'none'. NOTE: the `/users` endpoint will return an error unless you've preconfigured the schema in that external database.
+
+
+### Example Explained
```

Comment:
I'd add one more section under the "explained" header that shows where the fixtures are and refers to the TypeORM docs.